tests/v1/test_blog_update.py

```diff
@@ -9,44 +15,46 @@
 # Add the project root directory to the Python path
 sys.path.append(str(Path(__file__).resolve().parent.parent.parent))
 
-from main import app
-from api.utils.dependencies import get_db, get_current_user
-from api.v1.models.user import User
-from api.v1.models.blog import Blog
-from api.v1.schemas.blog import BlogRequest
-from fastapi.encoders import jsonable_encoder
```

Comment:
you made changes to a different file?

if not, pull from dev branch and make sure your branch is up to date

Reply:
They were not changes, the test files kept giving an error so I wanted to know what was wrong